src/ts/typings/types.d.ts

```diff
@@ -1,3 +1,5 @@
+import { Writable } from 'svelte/store';
+
```

Comment:
This will break the `declare` block, use inline imports instead.

Reply:
Fixed in e378179a506cf60e94a1a5764354f21ea2fa0b4b